lib/utils/generatepaymentqr.ts

```diff
@@ -0,0 +1,19 @@
+import { PaymentDetails } from "../types";
+import { generatePaymentLink } from "./generatepaymentlink";
+
+var QRCode = require('qrcode')
```

Comment:
Use ES6 `import` not require.

Reply:
@TomasGonzalez The ES6 import isn't working as TypeScript isn't recognizing the package through import statement(hence shows variable undefined error on testing) but it's working with require method.

Reply:
Did you tried to install the types using `npm install -D @types/module-name`

Reply:
yeah, it says:
`'QRCode' only refers to a type, but is being used as a value here.`

I replicated the method as defined in the package documentation.

Reply:
did you try to do `import * as Module from 'module'` after adding `@types/qrcode` ?

Reply:
yeah, it's not recognizing the import in the code.